lib/dynamic_type/src/dynamic_type/dynamic_type.h

```diff
@@ -69,6 +69,23 @@ struct Containers {
       std::type_identity<MemberTypes>...,
       std::type_identity<Templates<DynamicType>>...>;
 
+#if __cplusplus < 202002L
+  template <typename DynamicType, typename... MemberTypes>
+  using TypeIdentitiesAsTupleMaybeNoContainer =
+      TypeIdentitiesAsTuple<DynamicType, MemberTypes...>;
+#else
+  // Removing containers from type list so that it does not have issue with
+  // C++20 concepts in the containers of standard library. I believe this is
+  // a compiler bug:
+  // https://gcc.gnu.org/bugzilla/show_bug.cgi?id=111316
+  // https://gcc.gnu.org/bugzilla/show_bug.cgi?id=111504
+  // https://github.com/llvm/llvm-project/issues/67056
+  template <typename DynamicType, typename... MemberTypes>
+  using TypeIdentitiesAsTupleMaybeNoContainer = std::tuple<
```

Comment:
What's the implication of this change? Does it mean `DynamicType` cannot have containers?